src/ui/components/Button/index.js

```diff
@@ -6,30 +6,62 @@ import PropTypes from 'prop-types';
 import Link from 'amo/components/Link';
 import log from 'core/logger';
 
+import './styles.scss';
 
-import './Button.scss';
+
+const BUTTON_TYPES = [
+  'neutral',
+  'light',
+  'action',
+  'cancel',
+  'confirm',
+  'alert',
+  'none',
```

Comment:
Is this really necessary? Can it just be considered `none` if `type` is undefined?

Reply:
Yeah, that's fair. This started to make sure I caught any errant buttons in the conversion that didn't have a type. I think going forward an empty type meaning none is fine 👍